test/e2e/suite_test.go

```diff
@@ -518,6 +520,45 @@ func createSingleDatacenterCluster(t *testing.T, ctx context.Context, namespace
 		return cassandraDatacenterReady(kdcStatus.Cassandra)
 	}, polling.k8ssandraClusterStatus.timeout, polling.k8ssandraClusterStatus.interval, "timed out waiting for K8ssandraCluster status to get updated")
 
+	// Confirm default images are being applied as expected.
+	expectedSystemLoggerContainerImage := &images.Image{
+		Registry:   "docker.io",
+		Repository: "k8ssandra",
+		Name:       "system-logger",
+		Tag:        "latest",
+	}
+	expectedConfigBuilderContainerImage := &images.Image{
+		Registry:   "docker.io",
+		Repository: "datastax",
+		Name:       "cass-config-builder",
+		Tag:        "1.0.4-ubi7",
+	}
+	require.Equal(t, k8ssandra.Spec.Cassandra.SystemLoggerContainerImage, expectedSystemLoggerContainerImage)
+	require.Equal(t, k8ssandra.Spec.Cassandra.ConfigBuilderContainerImage, expectedConfigBuilderContainerImage)
+
+	// Confirm non-default images are being applied as expected and merged with defaults.
+	partialSystemLoggerContainerImage := &images.Image{
+		Tag: "latest",
```

Comment:
Don't we want something other than `latest` here since that is the default?

Reply:
Yes, you're entirely right. I was half way through updating when my laptop crashed. Docker on Mac! 😩